bootstrap/README.md

```diff
@@ -0,0 +1,11 @@
+# Bootstrap from MLOpsPython repository
+
+To bootstrap from the existing MLOpsPython repository run bootstrap.py script as below
+>python bootstrap.py --d [dirpath] --n [projectname]
```

Comment:
Explain what `dirpath` and `projectname` arguments are and requirements.  i.e. dirpath must be empty.